lib/public/components/lhcFillsDetail/index.js

```diff
@@ -31,6 +31,11 @@ const activeFields = (_model) => ({
         name: 'Fill number',
         visible: true,
     },
+    lhcPeriod: {
```

Comment:
`lhcPeriod` should be together with the rest of LHC data
Under `BetaStar`

Reply:
Moved the value to under the beta star variable.